src/drivers/Kconfig

```diff
@@ -35,4 +35,15 @@ config IPC_POLLING
 
 	  If unsure, select "n".
 
+config IMX_IRQSTEER
+	bool "Enable i.MX IRQ_STEER driver support"
+	default n
+	depends on IMX8 || IMX8X || IMX8M
```

Comment:
I think this should be placed under src/drivers/imx/Kconfig. Also, IMX_EDMA needs to be moved there.